sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CasesView.java

```diff
@@ -347,6 +347,9 @@ private void addCommonCasesOverviewToolbar() {
 						ImportExportUtils.getCaseExportProperties(caseFollowUpEnabled, hasCaseManagementRight),
 						CaseDownloadUtil::getPropertyCaption,
 						customExportWindow::close);
+					customExportsLayout.setExportCallbackPublicCustomConfiguration(
```

Comment:
this callback is the same as the one below, so there is no need for an extra callback.